modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -58,7 +58,7 @@ You can import files into your project, either separately or bundled together in
 
 == What to do next
 
-You can publish the API fragment to Anypoint Exchange. Click the *Publish to Exchange* button that is in the top-right corner of the code editor to open the *Publish API specification to Exchange* dialog.
+You can publish the API fragment to Anypoint Exchange. Click the *Publish* button that is in the top-right corner of the code editor, and then click *Publish to Exchange* to open the *Publish API specification to Exchange* dialog.
```

Comment:
```suggestion
You can publish the API fragment to Anypoint Exchange. Click *Publish* in the top-right corner of the code editor, and then click *Publish to Exchange* to open *Publish API specification to Exchange*.
```

Reply:
I'm also a little nervous that this is a "hidden" task, because we aren't using step format. Your best judgment, as always....